package.json

```diff
@@ -20,7 +20,11 @@
   "license": "OGL-UK-3.0",
   "dependencies": {
     "@hapi/hapi": "21.3.2",
-    "applicationinsights": "2.8.0"
+    "applicationinsights": "2.8.0",
+    "pg": "8.7.3",
+    "pg-hstore": "2.3.4",
```

Comment:
Incidentally, `pg-hstore` is required as a dependency of `sequelize` so does need to be included.

https://sequelize.org/docs/v6/getting-started/

![image](https://github.com/DEFRA/ffc-sfd-permissions-api/assets/42574977/357f9cf3-5662-4d10-8fb6-4c4def08fd8b)